submissions/april-1st/submission-april-1st.md

```diff
@@ -0,0 +1,208 @@
+
+(GUI image: easy)
+
+
+
+**Choleric cars**
+
+by [Therenas](https://mods.factorio.com/user/Therenas)
+
+Did your car/tank just crash into rock? Yeah it sucks. [This sound mod](https://mods.factorio.com/mod/cholericcars) lets the engineer express his opinion about it very clearly (sometimes with expletives). It definitely captures the feeling of the moment, often comedically well!
+
+**(image: hard)**
+
+
+
+**More sound mods**
+
+There are many more little sound mods out there that are specific references. For example, Command & Conquer fans might like this pack of [Red Alert sounds](https://mods.factorio.com/mod/Command_and_Conquer_RedAlert_Sounds) for programmable speakers and viewers of Trupen might enjoy a custom [rock mining](https://mods.factorio.com/mod/TrupenRocks) sound.
+
+
+
+
+
+### Crazy Combat
+
+
+
+**MLG FactOreo**
+
+by [Kiplacon](https://mods.factorio.com/user/Kiplacon)
+
+If combat in Factorio is too bland for you these days and you want to summon the essence of Major League Gaming montages, with all their dubstep-rich glory and dank memes (and some expletives) look no further. [This mod delivers](https://mods.factorio.com/mod/MLG_FactOreo). You can craft a sniper rifle for your no-scope pwnage and also craft meme magazines to deal some aptly named "dank damage" with any bullet-firing weapon, including turrets and cars. 
+
+**(1 good combat gif: hard)**
+
+
+
+**Train Gun**
+
+by [Twu.4.25](https://mods.factorio.com/user/Twu.4.25)
+
+Alright, perhaps you are more interested in a weapon that is more... Factorio-style. We all know that the ultimate killer in the game is no mere biter, but the mighty train! Armed with this knowledge, [this mod](https://mods.factorio.com/mod/traingun) delivers the crazy lategame superweapon that is the train gun. Unlocked after nuclear missiles, this powerful gun launches nuclear-fueled locomotives that furiously spin and tear through hoardes of enemies and rocks alike, before exploding magnificently! 
+
+**(1 good combat gif: easy)**
+
+
+
+### Wonka Level Logistics: Renai Transportation
+
+by [Kiplacon](https://mods.factorio.com/user/Kiplacon)
+
+
+
+And for last on this mod list we have saved the largest and most imaginative. This [amazing mod](https://mods.factorio.com/mod/RenaiTransportation) introduces a whole **world** of new transportation options that can liven up your factory with so much air time.
```

Comment:
You could also add something like: "Even though this mod is chaotic, it can still be used for beautiful bases. Have a look what the famed creator of [Josef](https://alt-f4.blog/ALTF4-39/) came up with. Something he dubbed 'The Dolphin base'"

It's always fun to shoutout players :)

Reply:
I also saw this iron smelting setup using throwers someone posted on youtube https://www.youtube.com/watch?v=vZOnYDQOXKU